modules/systemSettings.js

```diff
@@ -0,0 +1,304 @@
+function SystemSettings() {
+
+  this.systemsettings = {
```

Comment:
I'm not certain that we need default values for system settings. Our other modules have defaults because they're FFOSBR-specific, which means that if we hadn't stored them, we get to choose our own reasonable values.

System settings are part of the OS, though. This means that if we don't know what the user had done before, we can just grab whatever's currently set on the device, unless I'm missing something?